arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1325,6 +1328,160 @@
         }
       }
     },
+    "RoutingProperties": {
+      "description": "The routing related properties of the IoT hub. See: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messaging",
+      "type": "object",
+      "properties": {
```

Comment:
anything required here? Based on the description of the fallbackRoute, looks like it needs to be provided, as that will govern the behavior if none of the routing rule conditions are met from the rules provided in the routes array?